odoo/tools/misc.py

```diff
@@ -802,6 +809,44 @@ def deco(*args, **kwargs):
                 return func(*args, **kwargs)
         return deco
 
+    def emit(self, record):
+        pass
+
+
+class quiet_logger(logging.Handler):
+    """Temporary lower the max logging level.
+    """
+    def __init__(self):
+        self.old_handlers = None
+        self.old_propagate = None
+        self.had_error_log = False
+        super().__init__()
+
+    def __enter__(self):
+        logger = logging.getLogger()
+        self.old_handlers = logger.handlers[:]
```

Comment:
Since this is manipulating the handlers directly, it could just `logger.handlers.insert(0, self)` no?